### ISSUE TITLE
bug

### ISSUE THREAD

**Issue by fenneishi** [PeterL1n/BackgroundMattingV2#196]
https://github.com/PeterL1n/BackgroundMattingV2/blob/220003663d72b9bab207398217680ea1d22db0fe/train_refine.py#L262
 kornia.resize(pred_pha_sm, true_pha_lg.shape[2:]).sub(true_pha_lg).abs()) ---> kornia.resize(pred_pha_sm.detach(), true_pha_lg.shape[2:]).sub(true_pha_lg).abs())
